fawltydeps/types.py

```diff
@@ -155,6 +157,12 @@ def __post_init__(self) -> None:
         elif self.path.match("__pypackages__/?.*/lib"):
             return  # also ok
 
+        # Support Windows projects
+        if platform.system() == "Windows" and self.path.match(
+            os.path.join("Lib", "site-packages")
+        ):
+            return  # also ok
+
```

Comment:
Same as in an earlier comment, I'd prefer to co-locate the Windows code with the corresponding POSIX code (from line 152):

```python
        # Support vitualenvs, poetry2nix envs, system-wide installs, etc.
        if sys.platform.startswiith("win"):
            if self.path.match("Lib\\site-packages"):
                return  # all ok
        else:
            if self.path.match("lib/python?.*/site-packages"):
                if (self.path.parent.parent.parent / "bin/python").is_file():
                    return  # all ok

        # Also support projects using __pypackages__ from PEP582:
        if self.path.match("__pypackages__/?.*/lib"):
            return  # also ok
```

Reply:
Applied in e16b180234eb87b7d44db2dc2e822b7a92d6cc1e.